ddtrace/ddtrace.go

```diff
@@ -119,3 +119,9 @@ type StartSpanConfig struct {
 	// then this will also set the TraceID to the same value.
 	SpanID uint64
 }
+
+// Logger implementations are able to log given messages that the tracer might output.
+type Logger interface {
+	// Log prints the given message.
+	Log(msg string)
```

Comment:
Whose job is it to put newlines into the output? Does it need to be inside `msg` or performed by the logger itself?
The stdlib `log` used by `defaultLogger` handles both cases, but a user-defined `Logger` might need to have it documented.

Reply:
I don't really understand what you're suggesting. This seems like the wrong place for your comment because it seems like you're addressing implementation details. But maybe I'm misunderstanding what you're trying to hint at.

Are you saying to not add the newlines in the default logger implementation? If so, that'd be weird.

If you are saying to document here that the newline is added, I think that also seems an odd place to do so.

Reply:
I removed all `\n` occurrences from everywhere. I didn't know that the default `log` package adds them. It was weird to have them to begin with. Resolving.